app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -30,17 +39,57 @@ const WeatherDescription = ({ baseDescription, weather }) => {
   return baseDescription;
 };
 
+export const AnticipatedPrecipitation = ({ weather }) => {
+  const precip = [];
+  for (let i = 0; i < weather.minutely.length; i += 1) {
```

Comment:
Is there any way to set an interval to perform this calculation on a minute timer or potentially finding out how to run the weather query on a minute timer? Not sure where the interval is for running the weather queries or if the query is just rerun on refresh

Reply:
The weather queries are cached for 5 minutes to prevent exhausted the allotted API requests for the token. You could use `dt` as a timestamp to filter outdated minutely records.